log.txt

```diff
@@ -0,0 +1,672 @@
+[Mon, 14 Mar 2022 11:14:18 GMT] [info] [<0.000.0>] pouchdb-server has started on http://0.0.0.0:10102/
```

Comment:
Please commit .gitignore and untrack this file

Reply:
Ok